blocks/embed/embed.js

```diff
@@ -92,14 +92,19 @@ const loadEmbed = (block, grandChilds, link) => {
       match: ['twitter'],
       embed: embedSocialPlugins,
     },
+    {
+      match: ['instagram'],
+      embed: embedSocialPlugins,
+      customStyle: 'border: 1px solid rgb(219, 219, 219);',
```

Comment:
Better to put styles in CSS. Suggestion - You can introduce a `type` parameter instead of `customStyle` , set it to instagram and then in CSS, define the instagram specific style.